soc/common/riscv-privileged/Kconfig

```diff
@@ -9,3 +9,6 @@ config RISCV_VECTORED_MODE
 	depends on RISCV_PRIVILEGED
 	help
 	  Should the SOC use vectored mode.
+
+config RISCV_HAS_CPU_IDLE
```

Comment:
`RISCV_HAS_CPU_IDLE` should be removed after 3.7 - it is not used anywhere in the code right now